services/core-api/app/api/mines/permits/permit/resources/permit.py

```diff
@@ -125,6 +126,13 @@ def post(self, mine_guid):
         if not mine:
             raise NotFound('There was no mine found with the provided mine_guid.')
 
+        identity = NOWApplicationIdentity.find_by_mine_guid(mine.mine_guid)
+        description = None
+        if identity:
```

Comment:
can you rename description to be more clear - "application_type_description" or something similar?